Task 62. Верстка экрана поиска

ypypyxa opened the pull request:
background_search_state.xml:
Добавлен бэкграунд для отображения статуса поиска(количества вакансий)

colors.xml:
Добавлен цвет для подсказки поля ввода поискового запроса

cursor_search_edit.xml:
Добавлен курсор кастомного цвета (синего)

dimens.xml:
Добавлено значение отступа 12dp

fragment_search.xml:
Сверстан экран поиска

ic_filter.xml:
Переименован для приведения к общему виду

search_background_rounded.xml:
Удален, т.к. есть такой же background_rounded_for_search_result.xml

strings.xml:
Добавлены строковые переменные

styles.xml:
Отредактирован стиль кнопки фильтров на экране поиска

ypypyxa committed:
Верстка экрана поиска

background_search_state.xml:
Добавлен бэкграунд для отображения статуса поиска(количества вакансий)

colors.xml:
Добавлен цвет для подсказки поля ввода поискового запроса

cursor_search_edit.xml:
Добавлен курсор кастомного цвета (синего)

dimens.xml:
Добавлено значение отступа 12dp

fragment_search.xml:
Сверстан экран поиска

ic_filter.xml:
Переименован для приведения к общему виду

search_background_rounded.xml:
Удален, т.к. есть такой же background_rounded_for_search_result.xml

strings.xml:
Добавлены строковые переменные

styles.xml:
Отредактирован стиль кнопки фильтров на экране поиска

5ir1us commented on app/src/main/res/layout/fragment_search.xml:
```diff
@@ -1,13 +1,215 @@
+            android:id="@+id/edit_search"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="@drawable/background_rounded_for_search_result"
```
Цвета поля Эдит текст  разные не соответствуют макету в темной теме

ypypyxa replied:
Исправлено

ypypyxa committed:
Исправление замечаний ревьюера

background_rounded_for_search_result.xml:
Изменен цвет в соответствии с белой и темной темами

ypypyxa committed:
Merge branch 'develop' into DP-Task-62

5ir1us commented:
🚀
